client-js/app/elements/labrad-registry.ts

```diff
@@ -73,7 +96,8 @@ export class LabradRegistry extends polymer.Base {
   selectedItem: string;
 
   _computeSelectedType(selectedIdx: number): string {
-    if (this.dirs && selectedIdx < this.dirs.length) {
+    var ofs = this.path.length > 0 ? 1 : 0;
```

Comment:
Is this like the offset for the ```...``` at the beginning of the list? If so, can we call this ```offset```?